com.meetfranz.Franz.json

```diff
@@ -0,0 +1,57 @@
+{
+    "app-id": "com.meetfranz.Franz",
+    "base": "org.electronjs.Electron2.BaseApp",
+    "base-version": "18.08",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "franz",
+    "separate-locales": false,
+    "rename-desktop-file": "franz.desktop",
+    "rename-icon": "franz",
+    "finish-args": [
+        "--share=ipc",
```

Comment:
It doesn't work under wayland?

Reply:
It's an electron app so I suspect not right now.